pom.xml

```diff
@@ -149,7 +149,6 @@ THE SOFTWARE.
       <dependency>
```

Comment:
You need to just remove the entire section IIUC

Reply:
@oleg-nenashev not sure I get what you mean, removing the whole `dependencyManagement` section?

If so, then either I'm missing something, or I disagree: this would be out of scope for this change, and there are 10+ other dependencies declared there. So it would look like much more involved/risky. Some things like the servlet-api override are not even declared in the parent pom, so really I must be missing your point.

Reply:
Just for `junit`, of course. The rest could be eventually removed as well (or moved to BOM), but this is a separate story